packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -30,6 +30,7 @@ export interface DebugModeOptions {
     showDebugMenu: boolean;
     checkFirmwareAuthenticity: boolean;
     coinjoinAllowNoTor: boolean;
+    transports: Parameters<typeof TrezorConnect['init']>[0]['transports'];
```

Comment:
import `ConnectSettings` + `ConnectSettings['transports']` ?

Reply:
This would work too because at the moment `Parameters<typeof TrezorConnect['init']>[0]['transports']` and `ConnectSettings['transports']` point to the same type as you correctly point out. 
I felt like I prefer more binding this field verbosely with TrezorConnect init method, because it is exactly how `debug.transports` is meant to be used. But I don't have any strong preference here and will gladly change this if it helps this review to get through :smile:

Reply:
Maybe ConnectSettings is nicer. Because in the end this is connect settings and what if we wanted to use this field as an input for lets say TrezorConnect.changeSettings method?

Reply:
it's nicer for my ocd :)

Reply:
8ae654b72f3f3cc9a5ed094357c04bcaa224ddba ok